modules/salt/pages/salt-rate-limiting.adoc

```diff
@@ -0,0 +1,61 @@
+[[salt.rate.limiting]]
+= Rate Limiting
+
+Salt is able to run commands in parallel on a large number of minions.
+This can potentially create large amounts of load on your infrastructure.
+You can use these rate-limiting parameters to control the load in your environment.
+
+These parameters are all configured in the [filename]``/etc/rhn/rhn.conf`` configuration file.
+
+[NOTE]
+====
+Salt commands that are executed from the command line are not subject to these parameters.
+====
+
+
+
+== Batching
+
+There are two parameters that control how actions are sent to clients, one for the batch size, and one for the delay.
+
+When the Salt master sends a batch of actions to the target minions, it will send it to the number of minions determined in the batch size parameter.
+As each minion completes, the action begins on a new minion after the specified delay period.
+
+Choosing a lower batch size will reduce system load and parallelism, but might reduce overall performance for processing actions.
+
+The batch size parameter sets the maximum number of clients that can execute a single action at the same time.
+Adjust the [systemitem]``java.salt_batch_size`` parameter.
+Defaults to 100.
+
+Increasing the delay increases the chance that multiple minions will have completed before the next action is issued, resulting in fewer overall commands, and reducing load.
+
+The batch delay parameter sets the amount of time, in seconds, to wait after an action is processed before beginning to process the action on the next minion.
```

Comment:
```suggestion
The batch delay parameter sets the amount of time, in seconds, to wait after a command is processed before beginning to process the command on the next minion.
```

Reply:
see my comment above in https://github.com/SUSE/doc-susemanager/pull/445/files#r282799570